source/getting-started/build-host-setup.rst

```diff
@@ -113,24 +113,22 @@ Nasm 2.11
 
 Openssl (latest)
 
-Download from |https://indy.fulgan.com/SSL| (the latest version:  |https://indy.fulgan.com/SSL/openssl-1.0.2-x64_86-win64.zip|)
+Download from |https://slproweb.com/products/Win32OpenSSL.html| (Win64 version:  |https://slproweb.com/download/Win64OpenSSL_Light-1_1_1h.exe|)

```

Comment:
Does it make sense to provide a standard page? https://wiki.openssl.org/index.php/Binaries

Reply:
I think it might be better.  If they change the URL, our page will stay same.

Reply:
I have pushed an patch for Openssl 1.1.1 tool to support in SBL.
update https://github.com/slimbootloader/slimbootloader/pull/927.  Currently we use 1.0.2 openssl and this change should not impact existing users.

I feel its better to recommend 1.1.1+ version while build. 
Latest version is supported in https://slproweb.com/products/Win32OpenSSL.html.

Reply:
Updated documentation to provide top level link where user can download the required openssl binaries. Also updated process to updated openssl_path.